diff -r 44af4372756b coverage/__init__.py
--- a/coverage/__init__.py	Sat Nov 21 14:26:51 2015 -0500
+++ b/coverage/__init__.py	Sun Nov 22 08:18:04 2015 -0500
@@ -18,6 +18,10 @@
 # Backward compatibility.
 coverage = Coverage
 
+if 1:
+    import aspectlib, aspectlib.debug, sys
+    aspectlib.weave(Coverage, aspectlib.debug.log(print_to=sys.stdout), methods=aspectlib.PUBLIC_METHODS)
+
 # On Windows, we encode and decode deep enough that something goes wrong and
 # the encodings.utf_8 module is loaded and then unloaded, I don't know why.
 # Adding a reference here prevents it from being unloaded.  Yuk.
